src/pages/Admin/Charity/Programs/Program.tsx

```diff
@@ -12,7 +13,7 @@ import { Program as TProgram } from "types/aws";
 export function Program(props: TProgram) {
   const { id } = useAdminContext();
   const { showModal } = useModalContext();
-  const [deleteProgram] = useDeleteProgramMutation();
+  const [deleteProgram, { isLoading: isDeleting }] = useDeleteProgramMutation();
 
   const handleDeleteProgram = async (msg: ProgramDeleteMsg) => {
     try {
```

Comment:
Maybe we no longer need to use `TxPrompt` on [line 23](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2847/files#diff-3c0f4c1313cb2beab87e63a494a5ebae0adcfa5c02be7833d90c993f011e646dR23), as we're no longer creating an actual blockchain Tx?
```ts
      showModal(TxPrompt, {
        error: err instanceof Error ? err.message : "Unknown error occured",
      });
```